rubix-core/src/main/java/com/qubole/rubix/core/utils/DummyClusterManagerMultinode.java

```diff
@@ -45,4 +45,15 @@ public ClusterType getClusterType()
     {
         return ClusterType.TEST_CLUSTER_MANAGER;
     }
+
+    @Override
+    public String getCurrentNodeName()
+    {
+        return getNodes().get(0);
+    }
+
+    public String locateKey(String key)
```

Comment:
this change is to fix the flaky test `verifyNonlocalRequestMetricIsReported`to ensure request to get mapped to the different node than the current one.